rosdep/python.yaml

```
@@ -5899,6 +5899,13 @@ python3-empy:
   openembedded: [python3-empy@meta-ros-common]
   rhel: ['python%{python3_pkgversion}-empy']
   ubuntu: [python3-empy]
+python3-evdev:
+  debian: [python3-evdev]
```

We should null out supported distros where this package is not available. I would add entries for jessie and stretch as they are recommended support for Kinetic and Melodic which are not EOL.
```suggestion
  debian:
    '*': [python3-evdev]
    jessie: null
    stretch: null
```